stdlib/public/core/Integers.swift.gyb

```diff
@@ -223,7 +223,7 @@ def operatorComment(operator, fixedWidth):
   /// Returns the remainder of dividing the first value by the second.
   ///
   /// The result of the remainder operator (`%`) has the same sign as `lhs` and
-  /// is less than `rhs.magnitude`.
+  /// is less than `abs(rhs)`.
```

Comment:
Strictly speaking, the first version is more correct, since it accounts for `Int.min`.